text/0004-new-pre-coordicide-consensus/0004-new-pre-coordicide-consensus.tex.md

```diff
@@ -0,0 +1,191 @@
++ Feature name: New Pre-Coordicide Consensus
++ Start date: 2020-02-10
++ RFC PR: [iotaledger/protocol-rfcs#0004](https://github.com/iotaledger/protocol-rfcs/pull/4)
+
+
+# Summary
+
+A change to the coordinator based protocol that should enable higher throughput of TPS and CTPS. It is achieved by doing the following changes:
```

Comment:
`throughput of TPS and CTPS` seems a bit redundant to me as TPS/CTPS are already throughputs. I'd just say `enable higher TPS and CTPS`.